### PR TITLE
Add CLI command to list and toggle feature flags

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add CLI command to list and toggle feature flags
| Type?             | new feature
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI tests green are enough since integration tests have been added for this new command
| Fixed ticket?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Review by PrestaEdit — DISMISSED** [review]

File: src/PrestaShopBundle/Command/FeatureFlagCommand.php [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * This CLI command allows to enable/disable a feature flag or to list all of them.
+ */
+class FeatureFlagCommand extends Command
+{
+    private const SUCCESS_RETURN_CODE = 0;
```
Comment (0x346e3730) [inline src/PrestaShopBundle/Command/FeatureFlagCommand.php:44]: Seeing this when there are consts in the Console component on 5.4 🥲

File: tests/Integration/PrestaShopBundle/Command/FeatureFlagCommandTest.php [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        /** @var EntityManagerInterface $entityManager */
+        $entityManager = $kernel->getContainer()->get('doctrine.orm.entity_manager');
+        $entityManager->persist($featureFlag);
+        $entityManager->flush();
+    }
+
+    public static function tearDownAfterClass(): void
+    {
+        parent::tearDownAfterClass();
+        //DatabaseDump::restoreTables(['feature_flag']);
```
Comment (0x346e3730) [inline tests/Integration/PrestaShopBundle/Command/FeatureFlagCommandTest.php]:
```suggestion
```

**Review by 0x346e3730 — CHANGES_REQUESTED** [review]
Just a little thing that has been forgotten otherwise lgtm

**Review by boherm — DISMISSED** [review]
LGTM!

**Review by matthieu-rolland — COMMENTED** [review]
lgtm, minus the dump comment

File: tests/Integration/PrestaShopBundle/Command/FeatureFlagCommandTest.php (hunk 2 above)
Comment (jolelievre) [reply tests/Integration/PrestaShopBundle/Command/FeatureFlagCommandTest.php]: Good call! I commented it for a test I'll fix that ASAP

**Review by matthieu-rolland — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Review by 0x346e3730 — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
@PrestaShop/qa-functional it's a CLI command with auto-tests, please confirm that we can merge

**Comment by Robin-Fischer-PS** [comment]
Hi @jolelievre @kpodemski !
It can be merged without QA indeed ;)
Thanks!

**Pull request merged**